cmd/govim/vimstate.go

```diff
@@ -89,6 +89,17 @@ type vimstate struct {
 	// lastProgressText points to the text in the most recently created progress
 	// popup.
 	lastProgressText *strings.Builder
+
+	// vimgrepPendingBufs contain buffers read during a vimgrep quickfix command,
+	// keyed by buffer number.
+	// The purpose is to avoid sending DidOpen/DidClose notifications to gopls
+	// for all files that doesn't match the pattern.
```

Comment:
Small nit. Phrasing needs a bit of a tweak here I think. AIUI we're trying to avoid sending DidOpen/DidClose notifications to gopls for files which contained no pattern match, correct? Saying "files that didn't (sic) match the pattern" is a bit ambiguous because it could be the filepath that is matching a pattern or its contents.

Reply:
Done